frontend/app/src/components/settings/explorers/Explorers.vue

```diff
@@ -11,21 +11,30 @@ const { explorers } = storeToRefs(store);
 
 const address = ref<string>('');
 const tx = ref<string>('');
+const block = ref<string>('');
 
-const txUrl = computed<string>(() => {
+const addressUrl = computed<string>(() => {
   const setting = get(explorers)[get(selection)];
-  return setting?.transaction ?? explorerUrls[get(selection)].transaction;
+  return setting?.address ?? explorerUrls[get(selection)].address;
 });
 
-const addressUrl = computed<string>(() => {
+const txUrl: ComputedRef<string | null> = computed(() => {
```

Comment:
where we do multiple `get(selection)` in the same composable do you think you can extract it to `const chain = get(selection)`